src/site/content/en/blog/coop-coep/index.md

```diff
@@ -23,27 +23,42 @@ origin_trial:
 feedback:
   - api
 ---
+{% Banner 'caution', 'body' %}
+
+**Updates**
+
+**October 15th, 2020**: `self.crossOriginIsolated` is available from Chrome 87.
+Reflecting that, `document.domain` is immutable when `self.crossOriginIsolated`
+returns `true`. `performance.measureMemory` ending its origin trial and is
+planned to be enabled by default in Chrome 88. Shared Array Buffer on Android
+Chrome will be available from Chrome 88.
+
+**September 1st, 2020**: COOP Reporting is behind the `#cross-origin-isolated` flag
+in Chrome 86.
```

Comment:
@agektmr you previously said "Added instructions" here but I'm not seeing any instructions in this post. Also, please use our standard phrasing `See [X](link)` rather than `Added instructions`.

Reply:
This is the change I made for this: https://github.com/GoogleChrome/web.dev/commit/de54c4300e3d5fd76f424eb83e339a2a68d2b8d3#diff-5afd2bd259d9f1c14af088410be752de83c1eb1a05e55e8fae8cd0bb0d1df7e3
I wanted to link to the line starting with "The COOP Reporting API in Chrome is available after version 86 with one of two conditions:" but unable to do it because it's part of summary.

Reply:
I tried to add `{: #coop-reporting}` at inside `{% DetailsSummary %}` but proved it doesn't work. Please don't worry. This isn't a big deal.

Reply:
Ah, OK. In that case I think we can just use a [named anchor](https://stackoverflow.com/a/10510353/1669860). I'll add it now and see if it works.